src/css/_controls.scss

```diff
@@ -0,0 +1,256 @@
+$gray13: #222;
+$gray20: #333;
+$gray40: #666;
+$gray80: #ccc;
+$white: #fff;
+$controlsText: $white;
+$controlsBG: $gray20;
+$iconFill: $white;
+$iconActiveFill: $gray20;
+$iconStroke: $white;
+$iconActiveStroke: $gray20;
+$tooltipBG: $gray20;
+$activeButtonBG: $white;
+$controlsBorder: $gray13;
+$trackColor: $gray40;
+$trackFillColor: $gray80;
+
+.BRcontrols {
+  position: relative;
+  z-index: 7;
+  width: 100%;
+  background: $controlsBG;
+  button {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    padding: 0;
+    cursor: pointer;
+    border: none;
+    border-radius: 2px;
+    background: transparent;
+    outline: none;
+    &:hover .tooltip {
+      display: block;
+    }
+  }
+
+  .tooltip {
+    display: none;
+    position: absolute;
+    width: auto;
+    left: 50%;
+    bottom: calc(100% + 10px);
+    transform: translateX(-50%);
+    padding: 5px 10px;
+    font-weight: bold;
+    white-space: nowrap;
+    color: $controlsText;
+    border-radius: 3px;
+    background: $tooltipBG;
+  }
+
+  .full .tooltip {
+    left: auto;
+    right: 0;
+    transform: translateX(0);
+  }
+
+  .BRfullscreenActive & {
+    .icon-fullscreen {
+      background-image: url("icons/fullscreen_exit.svg");
+    }
+  }
+
+  select.playback-speed {
+    width: 24px;
+    padding-top: 18px;
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    font-size: 10px;
+    text-align: center;
+    text-align-last: center;
+    color: $controlsText;
+    border: none;
+    cursor: pointer;
+    background: transparent url("icons/playback-speed.svg") 50% 0 no-repeat;
+  }
+
+  .active {
+    background: $activeButtonBG;
+  }
+
+  li {
+    position: relative;
+  }
+
+  .controls {
+    position: relative;
+    z-index: 2;
+    display: flex;
+    padding: 2px 10px;
+    margin: 0;
+    list-style: none;
+    border-top: 1px solid $controlsBorder;
+    background: $controlsBG;
+  }
+
+  .controls button {
+    width: 30px;
+    height: 30px;
+  }
+
+  .read-aloud {
+    display: flex;
+    position: absolute;
+    bottom: -80px;
```

Comment:
what does this do?

Reply:
Positions the read aloud toolbar out of view behind the main toolbar until it's been enabled.